CMS/settings.py

```diff
@@ -39,6 +39,7 @@
     'django.contrib.staticfiles',
 # Adding djangorestframework to the poject
     'rest_framework',
+    'blog',
```

Comment:
Maybe a comment to document what you are adding. 

```suggestion
    'blog',
    #adding blog app to the manifest.
```